services/src/handlers/workflows.rs

```diff
@@ -59,7 +59,40 @@ where
     tag = "Workflows",
     post,
     path = "/workflow",
-    request_body = Workflow,
+    request_body(content = Workflow, examples(
+        ("MockPointSource" = (value = json!({
+            "type": "Vector",
+            "operator": {
+                "type": "MockPointSource",
+                    "params": {
+                        "points": [
+                            { "x": 0.0, "y": 0.1 },
+                            { "x": 1.0, "y": 1.1 }
+                        ]
+                    }
+                }
+        }))),
+        ("Statistics Plot" = (value = json!({
+            "type": "Plot",
+            "operator": {
+                "type": "Statistics",
+                "params": {},
+                "sources": {
+                    "source": {
+                        "type": "OgrSource",
+                        "params": {
+                            "data": {
+                                "type": "internal",
+                                "datasetId": "c36f5ce7-d0df-4982-babb-cc9e67d2a196"
```

Comment:
It says 
```json
{
  "error": "Operator",
  "message": "Operator: LoadingInfoError: Operator: DatasetMetaData: UnknownDatasetId"
}
```

Reply:
This is expected if the dataset id is non existend. This is just an example on how one would create a dataset using OgrSource. The user needs to have a dataset first which he can reference. This is why in the plot example the user is first instructed to upload some data.